File: MODUL 4/PRAK402-2410817210005-Muhammad_Irgi_Fahrezha.py
batas = int(input())
i = 1

while i <= batas:
    if i % 2 != 0:
        print(i, end=" ")
    i += 1

print("")

while batas >= 1:
    if batas % 2 == 0:
        print(batas, end=" ")
    batas -= 1

File: MODUL 4/PRAK402-2410817210005-Muhammad_Irgi_Fahrezha.c
#include <stdio.h>

int main () {

    int batas;
    int i = 1;

    scanf("%d", &batas);

    while (i <= batas) {
        if (i % 2 != 0) {
            printf("%d ", i);
        }
        i++;
    }
    printf("\n");

    while (batas >= 1) {
        if (batas % 2 == 0) {
            printf ("%d ", batas);
        }
        batas--;
    }

    return 0;
}